pages/configuring_webif.rst

```diff
@@ -182,6 +182,31 @@ If you want to use nginx to proxy access to a Graylog server, you have several o
     }
   }
 
+Apache:
+------- 
+
+For the following samples we are assuming that your Graylog instance is running on ``graylog.internal.example.org`` using the default ports of 12900 for the REST API and 9000 for the web interface. SSL is disabled for both. You want to expose the Graylog web interface as ``https://graylog.example.org``. 
+
+**URI Configs in Graylog server conf**::
+
+   rest_listen_uri = http://127.0.0.1:12900/
+   web_listen_uri = http://127.0.0.1:9000/
+   web_endpoint_uri = https://graylog.example.org/api/
+ 
+**REST API and Web Interface on one port (using HTTPS/SSL)**::
+   Listen 443
+   <VirtualHost *:443>
+       ServerName graylog.example.org
+       Your SSL config
```

Comment:
Please add a comment sign to make clear that this needs to be modified after copy & pasting :)